pkg/cache/cache.go

```diff
@@ -110,19 +110,19 @@ func (c *Client) New(log *logger.Log, isRedis bool, host, pass, database string)
 
 // Delete delete decision in cache.
 func (c *Client) Delete(key string) {
-	c.log.Debug(fmt.Sprintf("cache:Delete key:%v", key))
+	// c.log.Debug(fmt.Sprintf("cache:Delete key:%v", key))
```

Comment:
This is more like trace rather than debug? I can't imagine trying to troubleshoot something on a bussy cluster with this log enabled.

Reply:
In a production/bussy cluster, you don't enable DEBUG log ?

Reply:
@maxlerebourg usually no, loglevel in production should be error or info

Reply:
> loglevel in production should be error or info

I totally agree with this, until you have a fire that cannot be reproduced outside of the production environment easily or in a timely manner. Then it can come in very handy. I introduced a new Trace log level in the PR, which would allow to emergency enable debug if needed, without flooding the logs.